### PR TITLE
TMEDIA-176 - Add logic for video ANS type

### PR DESCRIPTION
**Pull request by matthewroach**
## Description

Add logic to account for video type ANS objects

## Jira Ticket
- [TMEDIA-176](https://arcpublishing.atlassian.net/browse/TMEDIA-176)

## Test Steps

1. Checkout this branch `git checkout TMEDIA-176-collection-resizer-with-video`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/resizer-image-block`
3. Visit test page - and verify images show for video stories - http://localhost/pf/test-tmedia-176/?_website=the-gazette

### Before

Images are not showing for video's

TMEDIA-176-before

### After

Images show for video's

TMEDIA-176-after

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [X] Confirmed all the test steps a reviewer will follow above are working.
- [X] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [X] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [X] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [X] Confirmed this PR has unit test files
  - [X] Ran `npm run test`, made sure all tests are passing
- [X] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/resizer-image-block/getResizedImageData.test.js [hunk 1]
```diff
@@ -520,6 +521,29 @@ describe('get resized image data helper on the server-side', () => {
...
+        mockCollectionApiData,
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+      );
+
+      expect(typeof dataSearch.content_elements[0].promo_items.basic.resized_params).toEqual(
+        'object',
```
Comment (JackHowa) [inline blocks/resizer-image-block/getResizedImageData.test.js:544]: not sure I wildly follow this test case. did this fail before? would it have returned null if it wasn't working?

**Comment by JackHowa** [comment]
will pull this down locally

**Review by JackHowa — APPROVED** [review]
working as expected locally; good quick test case visually -- nice

Screen Shot 2021-04-12 at 10 55 38

File: blocks/resizer-image-block/getResizedImageData.test.js (hunk 1 above)
Comment (matthewroach) [reply blocks/resizer-image-block/getResizedImageData.test.js:544]: This is a new test case, and wouldn't return before

**Pull request merged**